code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -41,6 +41,8 @@
 
 	///If there is an active hotspot on us store a reference to it here
 	var/obj/effect/hotspot/active_hotspot
+	///the group of hotspots we are a part of
+	var/datum/hot_group/our_hot_group
```

Comment:
surely this should be something hotspots manage? if a hotspot is deleted does the group persist?